sys/fs/fuse/fuse_vnops.c

```diff
@@ -2108,10 +2110,10 @@ fuse_vnop_remove(struct vop_remove_args *ap)
 	int err;
 
 	if (fuse_isdeadfs(vp)) {
-		return ENXIO;
+		return (EXTERROR(ENXIO, "This FUSE session is about to be closed"));
 	}
 	if (vnode_isdir(vp)) {
-		return EPERM;
+		return (EXTERROR(EPERM, "vnode is a directory"));
```

Comment:
Huh.  I would've though that his should be EISDIR.  But I checked just now, and opengroup does specify EPERM.  It's also probably unreachable.